roles/etcd/tasks/main.yml

```diff
@@ -28,9 +31,6 @@
   when: etcd_deployment_type == "host" and etcd_copy.stdout_lines and is_etcd_master
     or etcd_secret_changed|default(false)
 
-# Reload systemd before starting service
```

Comment:
Let's flush twice, actually. L28 triggers a restart if certs were changed and we need that done before L37